packages/@react-spectrum/sidenav/src/SideNav.tsx

```diff
@@ -20,10 +20,10 @@ import styles from '@adobe/spectrum-css-temp/components/sidenav/vars.css';
 import {TreeState, useTreeState} from '@react-stately/tree';
 import {useSideNav, useSideNavItem} from '@react-aria/sidenav';
 
-export interface SideNavProps<T> extends CollectionBase<T>, SingleSelectionBase, Expandable, StyleProps {}
+export interface SideNavProps<T> extends CollectionBase<T>, MultipleSelection, Expandable, StyleProps {}
```

Comment:
Why the change from `SingleSelection` to `MultipleSelection`? Is it because SideNav will support multi selection in the future?

Reply:
@LFDanLu I'll need little help with this... SIdeNav is using useTreeState suggested from ticket https://jira.corp.adobe.com/browse/RSP-1143 
like this:
https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-spectrum/sidenav/src/SideNav.tsx#L26

and useTreeState only works with MultipleSelection, SingleSelection would be incompatible interface... so I'm not sure whats smart thing to do :/

Reply:
hrm, I see, I'm guessing it's because useTreeState calls `useMultipleSelectionState`. In that case you can keep your stuff as is for now.

Maybe SingleSelection shouldn't even exist since MultipleSelection allows for strict single selection as well? Dunno

Reply:
SingleSelection will require writing a new hook, and it would act exactly as MultipleSelection with single selection mode

Reply:
Hm, ok, I think we can make this a follow up discussion or ignore it for now since it is just a type naming thing.